jaeger-crossdock/src/main/java/com/uber/jaeger/crossdock/api/CreateTracesRequest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2017, Uber Technologies, Inc
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.uber.jaeger.crossdock.api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.ToString;
+
+import java.util.Map;
+
+@ToString
+public class CreateTracesRequest {
+  private String operation;
+  private int count;
+  private Map<String, String> tags;
+
+  @JsonCreator
+  public CreateTracesRequest(
+      @JsonProperty("operation") String operation,
+      @JsonProperty("count") int count,
+      @JsonProperty("tags") Map<String, String> tags) {
+    this.operation = operation;
+    this.count = count;
+    this.tags = tags;
+  }
+
+  public String getOperation() {
```

Comment:
I missed this last time, but you could simply use `@Getter` on the class to generate getters for all private fields